topics/index.md

```diff
@@ -50,6 +51,7 @@ Administration
 * [Persistence](persistence.md): Options for configuring durability using disk backups.
 * [Administration](admin.md): Various administration topics.
 * [Security](security.md): An overview of Valkey's security.
+* [RDMA](RDMA.md#Getting-Started): An overview of RDMA support.
```

Comment:
The correct link is 'getting-started' in lowercase letters.

```suggestion
* [RDMA](RDMA.md#getting-started): An overview of RDMA support.
```

Maybe just link to RDMA.md full page?

You can see the link names, etc., if you click "View file" when looking at the diff for RDMA.md

![image](https://github.com/valkey-io/valkey-doc/assets/273886/dee61da9-50c8-4f7b-9cbc-cf27d5b605ff)

Reply:
Sure, leave a single entry for RDMA, so let this one link the full RDMA.md in the new version.